apps/generator/docs/generator-template.md

```
@@ -39,45 +39,64 @@ Before you begin, make sure you have the following set up:
       additionalProperties: false
       properties:
-        temperatureId:
+        value:
+          type: number
+        unit:
           type: string
+
 ```
+> 🛠️ **Note on Payload Schema Refactor:**  
+> n this version, the payload definition has been moved to a reusable schema component (#/components/schemas/Temperature). This change reflects AsyncAPI v3 best practices, but doesn't affect functionality. This aligns with AsyncAPI v3's best practices for reuse and separation of concerns. Only the naming changed (temperatureId → value) to more clearly match the data's purpose. This is purely cosmetic and doesn’t affect function.
+
```

markdownlint-cli2 (0.17.2)

Blank line inside blockquote
null

(MD028, no-blanks-blockquote)